### ISSUE TITLE
Run  playground in container

### ISSUE THREAD

**Issue by Ehco1996** [pingcap/tiup#1430]
## Feature Request

**Is your feature request related to a problem? Please describe:**

as title

**Describe the feature you'd like:**

run tidb playground in one container

**Why the featue is needed:**

Many individual developers cannot afford the high price of the TiDB-cluster, but they may want to use related tools from the TiDB community.

For example, the TiDB dashboard / one-click Grafana monitor etc.

Consider that tiup already provides "tiup --tag xxx playground" to ensure that data is not lost.

Perhaps we could go a step further and provide docker images to help individual developers replace their mysql with tidb playground ?

By the way I think we can think of TiDB-playground as a lighter version of TiDB-cluster, e.g. the relationship between K3s and K8s

**Describe alternatives you've considered:**

I found this [repo](https://github.com/yeya24/tidb-playground), which may help a little

**Teachability, Documentation, Adoption, Migration Strategy:**

**Comment by AstroProfundis** [comment]
No, `tiup-playground` is designed to be "played" and so it should be as easy as possible but stability is not a must, so involving dependency of container layer is not suitable for this goal.

But it's definitely a good idea to have something working with simple k8s environment to let users try TiDB on cloud easily, I'd prefer that to be a new, dedicated component (instead of inside of `tiup-playground`) to do so.
